pkg/webui/account/containers/sessions-table/index.js

```diff
@@ -0,0 +1,219 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { connect, useDispatch } from 'react-redux'
+import { defineMessages } from 'react-intl'
+import { bindActionCreators } from 'redux'
+
+import Button from '@ttn-lw/components/button'
+import Status from '@ttn-lw/components/status'
+import toast from '@ttn-lw/components/toast'
+
+import FetchTable from '@ttn-lw/containers/fetch-table'
+
+import Message from '@ttn-lw/lib/components/message'
+import DateTime from '@ttn-lw/lib/components/date-time'
+
+import PropTypes from '@ttn-lw/lib/prop-types'
+import sharedMessages from '@ttn-lw/lib/shared-messages'
+import attachPromise from '@ttn-lw/lib/store/actions/attach-promise'
+
+import { getUserSessionsList, deleteUserSession } from '@account/store/actions/user'
+
+import {
+  selectUserId,
+  selectUserSessions,
+  selectUserSessionsTotalCount,
+  selectUserSessionsFetching,
+  selectSessionId,
+} from '@account/store/selectors/user'
+
+const m = defineMessages({
+  deleteSessionSuccess: 'Session removed successfully',
+  deleteSessionError: 'There was an error and this session could not be deleted',
+  sessionsTableTitle: 'Sessions',
+  removeButtonMessage: 'Remove this session',
+  noExpiryDate: 'No expiry date',
+  endSession: 'Logout to end this session',
+  currentSession: '(this is the current session)',
+})
+
+const UserSessionsTable = props => {
+  const { selectTableData, pageSize, user, handleDeleteSession } = props
+  const dispatch = useDispatch()
+
+  const getSessions = React.useCallback(filters => getUserSessionsList(filters, user), [user])
+
+  const deleteSession = React.useCallback(
+    async session_id => {
+      try {
+        await handleDeleteSession(session_id)
+        toast({
+          message: m.deleteSessionSuccess,
+          type: toast.types.SUCCESS,
+        })
+        dispatch(getUserSessionsList(user))
+      } catch {
+        toast({
+          message: m.deleteSessionError,
+          type: toast.types.ERROR,
+        })
+      }
+    },
+    [user, handleDeleteSession, dispatch],
+  )
+
+  const makeHeaders = React.useMemo(() => {
+    const baseHeaders = [
+      {
+        name: 'session_id',
+        displayName: sharedMessages.id,
+        width: 9,
+      },
+      {
+        name: 'status',
+        displayName: '',
+        width: 17,
+        render: status => {
+          if (status.currentSession) {
+            return <Status status="none" label={m.currentSession} />
```

Comment:
I get the idea of using `status="none"` but there is no need to add this, because you can just as well use a regular `<Message />` instead. The `<Status />` component only makes sense to be used when we want to show a status as a color dot, which we don't want to in this case.